coalib/processes/BearRunning.py

```diff
@@ -408,13 +397,27 @@ def run_local_bears(filename_queue,
     try:
         while True:
             filename = filename_queue.get(timeout=timeout)
+
+            if filename not in file_dict:
+                send_msg(message_queue,
+                         timeout,
+                         LOG_LEVEL.ERROR,
+                         "An internal error occurred.",
+                         Constants.THIS_IS_A_BUG)
```

Comment:
can you make a commit removing this whole if? This is an internal error and should never occurs we can just let it throw the exception :)